src/components/calcite-dropdown-item/calcite-dropdown-item.tsx

```diff
@@ -122,13 +122,25 @@ export class CalciteDropdownItem {
         {slottedContent}
       </a>
     );
+
+    const itemRole = this.href
```

Comment:
Nested ternaries are really hard for me to parse. I may be alone, but I  have to stare at these for a really long time to understand what's going on.

Reply:
I can... switch... that up, agree it's a little wordy.

Reply:
Updated to switch